ynr/apps/people/models.py

```diff
@@ -169,6 +169,8 @@ def get_value_html(self):
 
         if self.value_type == "twitter_username":
             url = format_html("https://twitter.com/{}", self.value)
+        if self.value_type == "instagram_url":
+            url = format_html("https://www.instagram.com/{}", self.value)
```

Comment:
Maybe I'm reading this wrong, but isn't `clean_instagram_url` returning a full URL? In that case, the url here should just be `self.value`?